<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="IMCOBCP97-MSL1" ignored="1" name="Default test(failed)" tests="2" failures="1" timestamp="31 Dec 2020 15:37:22 GMT" time="0.042" errors="0">
  <testcase name="test2" time="0.009" classname="testCases.TestNgRetryFailedDemo"/>
  <testcase name="test3" time="0.003" classname="testCases.TestNgRetryFailedDemo">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TestNgRetryFailedDemo.test3(TestNgRetryFailedDemo.java:24)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase name="test1" time="0.0" classname="testCases.TestNgRetryFailedDemo">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
